tools/cloud-build/daily-tests/ansible_playbooks/test-batch-submission.yml

```diff
@@ -0,0 +1,35 @@
+# Copyright 2022 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+---
+- name: Wait for startup script to complete
+  become: true
+  ansible.builtin.wait_for:
+    path: /var/log/messages
+    search_regex: '.*{{ remote_node }}.*startup-script exit status ([0-9]+)'
+    timeout: 600
+  register: startup_status
```

Comment:
There should be an additional task that fails out if the exit status is not 0. 
https://github.com/GoogleCloudPlatform/hpc-toolkit/blob/main/tools/cloud-build/daily-tests/ansible_playbooks/spack.yml#L25
I also noticed it's not checked for the monitoring test. I'll add a task to update that.

Reply:
Done.